docs/internals/release-process.txt

```diff
@@ -108,30 +138,29 @@ each version.
   regressions is much less of a concern.
 
 As a concrete example, consider a moment in time halfway between the release of
-Django 1.7 and 1.8. At this point in time:
+Django Y.0 and Y.1. At this point in time:
 
-* Features will be added to development master, to be released as Django 1.8.
+* Features will be added to development master, to be released as Django Y.1.
 
-* Critical bug fixes will be applied to the ``stable/1.7.x`` branch, and
-  released as 1.7.1, 1.7.2, etc.
+* Critical bug fixes will be applied to the ``stable/Y.0.x`` branch, and
+  released as Y.0.1, Y.0.2, etc.
 
 * Security fixes and bug fixes for data loss issues will be applied to
-  ``master`` and to the ``stable/1.7.x``, ``stable/1.6.x``, and
-  ``stable/1.4.x`` (LTS) branches. They will trigger the release of ``1.7.1``,
-  ``1.6.1``, ``1.4.1``, etc.
+  ``master`` and to the ``stable/Y.0.x``, and ``stable/X.2.x`` (LTS) branches.
+  They will trigger the release of ``Y.0.1``, ``X.2.1``, etc.
 
 * Documentation fixes will be applied to master, and, if easily backported, to
-  the ``1.7.x`` and ``1.6.x`` branches.
+  the latest stable branch, ``Y.0.x``.
 
 .. _lts-releases:
 
 Long-term support (LTS) releases
```

Comment:
I think this section should be moved up to immediately follow the discussion of the other release types (feature and security/bugfix). Otherwise there are a bunch of confusing forward-references to the LTS concept in the deprecation and semver sections.